VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/switch-on-camera-microphone/switch-on-camera-microphone.component.ts

```diff
@@ -72,15 +73,25 @@ export class SwitchOnCameraMicrophoneComponent extends ParticipantStatusBaseDire
     }
 
     async requestMedia() {
-        this.mediaAccepted = await this.userMediaStreamService.requestAccess();
-        this.userPrompted = true;
-        if (!this.mediaAccepted) {
-            this.logger.warn(`[SwitchOnCameraMicrophone] - ${this.participantName} denied access to camera.`, {
-                conference: this.conferenceId,
-                participant: this.participantName
+        this.userMediaStreamService.currentStream$
+            .pipe(first())
```

Comment:
I thought pipe can take multiple operators as arguments? So can we not do this: 
.pipe(first(), timeout(1000)) 

However I am not 100% sure you can do that and I will leave it upto you as I am not sure if there is any major benefits in using the alternative.

Reply:
It can, I think prettier left them both on one line so I did it using that to split it up so it was easier to read. I can change it back but I don't think there are any downsides.